src/formatters/Utils.cc

```diff
@@ -286,6 +286,24 @@ std::string formatFitsProperties(daf::base::PropertyList const& prop,
 int countFitsHeaderCards(lsst::daf::base::PropertySet const& prop) {
     return prop.paramNames(false).size();
 }
+
+ndarray::Array<std::uint8_t, 1, 1> stringToBytes(std::string const& str) {
+    auto nbytes = str.size() * sizeof(char) / sizeof(std::uint8_t);
+    std::uint8_t const* byteCArr = reinterpret_cast<std::uint8_t const*>(str.data());
```

Comment:
So we don't do any formal encoding? Hmm, I'm very worried that this will create problems when the data is read on a different platform then the one it was written.

Reply:
Discussed on the JIRA ticket with the resolution that we will file another ticket to look at this and use this code for now.